### ISSUE TITLE
Steam doesn't login

### ISSUE THREAD

**Issue by SpinalBlood** [miranda-ng/miranda-ng#781]
I cannot connect to Steam, no matter what I do, even with a new Miranda profile.  I can connect on the original website. I know at least another user that has the same problem

I tested in Windows 7 64 and Windows Xp 32 (Miranda always in 32bit)
In Windows Xp I get this message (no messages in Windows 7):

Warning:
SSL connection failure (80090326 377): Il messaggio ricevuto era imprevisto o con formattazione scorretta.

(it says the received message was unexpected or with wrong formatting)

[netlog.txt](https://github.com/miranda-ng/miranda-ng/files/879205/netlog.txt)

**Comment by aunsane** [comment]
Could you provide [verion info](http://wiki.miranda-ng.org/index.php?title=Version_information)?

**Comment by SpinalBlood** [comment]
CPU: Intel(R) Core(TM) i5-3450 CPU @ 3.10GHz [Intel64 Family 6 Model 58 Stepping 9] [DEP Enabled] [4 CPUs]
Installed RAM: 3792 MBytes
Microsoft Windows 7 Ultimate Edition, 64-bit Service Pack 1 (build 7601)
Internet Explorer: 9.11.9600.17801 (build 99600)
Administrator privileges: Yes
OS Languages: (UI | Locale (User/System)) : Italian/Italian | Italian/Italian
Free disk space on Miranda partition: 91557 MBytes

Miranda NG Version: 0.95.6 alpha build #17953 (75876f6) [running inside WOW64]
Build time: 22 Mar 2017 9:52:08
Profile: C:\Program Files (x86)\Miranda\Profiles\temp\temp.dat (dbx mmap driver)
Profile size: 131072 Bytes
Profile creation date: 22 Mar 2017 18:00:08
Language pack: No language pack installed
Service Mode: No

[b]Active Plugins (16):[/b]
¤ AdvaImg.dll v.[b]0.11.0.2[/b] [13 Jan 2017 5:01:19] - Miranda image services 
¤ AVS.dll v.[b]0.95.3.4[/b] [22 Mar 2017 9:47:36] - Avatar service 
¤ Clist_modern.dll v.[b]0.9.1.5[/b] [22 Mar 2017 9:49:22] - Modern contact list 
¤ CrashDumper.dll v.[b]0.0.5.2[/b] [13 Jan 2017 5:01:53] - Crash dumper 
¤ DbEditorPP.dll v.[b]4.0.1.1[/b] [4 Jan 2017 8:36:36] - Database editor++ 
¤ Dbx_mmap.dll v.[b]0.95.3.1[/b] [22 Mar 2017 9:47:40] - Miranda NG mmap database driver 
¤ HistoryPP.dll v.[b]1.5.1.5[/b] [9 Feb 2017 18:55:12] - History++ 
¤ Import.dll v.[b]0.95.4.3[/b] [18 Jan 2017 4:00:10] - Import contacts and messages 
¤ NewXstatusNotify.dll v.[b]1.4.0.11[/b] [18 Jan 2017 4:01:00] - xStatus notify 
¤ PluginUpdater.dll v.[b]0.1.2.5[/b] [13 Jan 2017 5:02:06] - Plugin updater 
¤ Popup.dll v.[b]2.1.1.12[/b] [18 Jan 2017 4:01:10] - Popup plus 
¤ SeenPlugin.dll v.[b]5.0.5.2[/b] [3 Dec 2016 4:01:06] - Last seen 
¤ StatusManager.dll v.[b]0.11.2.0[/b] [12 Feb 2017 4:04:14] - Status manager 
¤ Steam.dll v.[b]0.11.6.1[/b] [18 Jan 2017 4:03:58] - Steam protocol 
¤ TabSRMM.dll v.[b]3.5.0.4[/b] [22 Mar 2017 9:47:52] - TabSRMM 
¤ TopToolBar.dll v.[b]0.8.0.4[/b] [13 Jan 2017 5:01:35] - TopToolBar 

[b]Unloadable Plugins (1):[/b]
¤ DbChecker.dll v.[b]0.0.0.0[/b] [22 Mar 2017 9:49:38] - <unknown> 


[b]Protocols and Accounts:[/b]
-------------------------------------------------------------------------------
STEAM                    1 - Enabled 0 - Disabled  Loaded
MetaContacts             1 - Enabled 0 - Disabled  Loaded

[b]Icon Packs:[/b]
-------------------------------------------------------------------------------
 HistoryPP_icons.dll [10 Feb 2017 4:02:16]
 Proto_AIM.dll [22 Mar 2017 9:49:12]
 Proto_Facebook.dll [22 Mar 2017 9:49:10]
 Proto_GG.dll [31 Jan 2017 4:01:38]
 Proto_ICQ.dll [21 Feb 2017 4:01:40]
 Proto_IRC.dll [22 Mar 2017 9:49:12]
 Proto_Jabber.dll [22 Mar 2017 9:49:12]
 Proto_MetaContacts.dll [22 Mar 2017 9:51:20]
 Proto_MSN.dll [22 Mar 2017 9:49:12]
 Proto_Skype.dll [22 Mar 2017 9:51:34]
 Proto_Steam.dll [18 Jan 2017 4:03:52]
 Proto_Tox.dll [22 Mar 2017 9:51:36]
 Proto_Yahoo.dll [23 Sep 2016 3:01:40]
 Proto_YAMN.dll [14 Jan 2017 4:02:30]
 TabSRMM_icons.dll [22 Mar 2017 9:49:54]
 Toolbar_icons.dll [22 Mar 2017 9:49:12]
 Watrack_buttons.dll [18 Jan 2017 4:04:42]
 Watrack_icons.dll [18 Jan 2017 4:04:42]
 xStatus_ICQ.dll [21 Feb 2017 4:00:32]
 xStatus_Jabber.dll [22 Mar 2017 9:47:58]

**Comment by SpinalBlood** [comment]
Ok, I tried to register a new Steam account. Looks like I can connect on this new one.

I have tried to add some friends in the original website (just to be safe, if I recall there were some problems in Miranda doing that), to see if I can see them in Miranda later too. But looks like I can't because:

> _Having trouble accessing a Steam Community Feature?
New accounts are limited from using all of Steam's community features. You'll have access to all of Steam's features once you've spent $5.00 USD in the Steam store or added $5.00 USD to your Steam Wallet. Some of the features that you won't be able to access are:_

> _Adding friends on Steam
Buying or Selling items on the Steam Community Market
Creating a Group on Steam Community
Creating user reviews or rating artwork, screenshots, workshop content, or Greenlight submissions
Using Friends chat in your Browser or on a Mobile device_

So it looks like I'm reported as online, so it's probably working.

Why my main account can't connect, but I can on the new one?
